packages/neo-one-cli/src/InteractiveCLI.ts

```diff
@@ -263,18 +263,32 @@ export class InteractiveCLI {
 
             // tslint:disable-next-line no-any
             let spinner: any | undefined;
+            // tslint:disable-next-line no-any
+            let verSpinner: any | undefined;
+
             try {
-              const { pid } = await manager.start({
+              const { pid, newerVersion } = await manager.start({
                 port,
                 httpPort,
                 binary: createBinary(argv, this.serverConfig),
                 onStart: () => {
                   if (first) {
+                    verSpinner = ora(`Checking for @neo-one/server updates...`).start();
                     spinner = ora(`Starting ${name.title} server...`).start();
                   }
                 },
               });
 
+              if (verSpinner !== undefined) {
+                if (newerVersion === undefined) {
+                  verSpinner.succeed(`@neo-one/server is current`);
+                } else {
+                  verSpinner.warn(
+                    `A newer version (${newerVersion}) of @neo-one/server is available for download see https://neo-one.io/docs/playground#Updating or we can update the https://github.com/neo-one-suite/neo-one-playground to be self descriptive with installation and update commands as well as a link to the setup docs?`,
```

Comment:
This isn't for updating the playground, this is for updating the `neo-one` binary. This should be a section in https://neo-one.io/docs/environment-setup